.circleci/config.yml

```diff
@@ -78,7 +81,10 @@ workflows:
       - trigger-beta-build:
           requires:
             - prep-deps
-      - prep-deps
+      - prep-deps:
+          filters:
+            branches:
+              ignore: l10n_crowdin_action
```

Comment:
Hmm. Fair point that most of the runs on this branch aren't productive, but in theory we should be regularly merging this branch. And we'd want to ensure it passes CI before merging (we have seen CI catch errors caused by localization problems)

Maybe better to undo this change for now? We're introducing a new translations process soon that should eliminate this branch anyway, we can survive until then.

Reply:
Okay undone